var/spack/repos/builtin/packages/madgraph5amc/package.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class Madgraph5amc(Package):
+    """MadGraph5_aMC@NLO is a framework that aims at providing
+       all the elements necessary for SM and BSM phenomenology,
+       such as the computations of cross sections, the generation
+       of hard events and their matching with event generators,
+       and the use of a variety of tools relevant to
+       event manipulation and analysis. """
+
+    homepage = "https://launchpad.net/mg5amcnlo"
+    url      = "https://launchpad.net/mg5amcnlo/2.0/2.7.x/+download/MG5_aMC_v2.7.3.tar.gz"
+
+    version('2.7.3.py3', sha256='400c26f9b15b07baaad9bd62091ceea785c2d3a59618fdc27cad213816bc7225')
```

Comment:
I think you need to specify a URL for this version too.

Reply:
As far as I can tell, spack can properly determine the URL to 2.7.3.py3 version

Reply:
What's the output of `spack info madgraph5amc`?

Reply:
```
Package:   madgraph5amc

Description:
    MadGraph5_aMC@NLO is a framework that aims at providing all the elements
    necessary for SM and BSM phenomenology, such as the computations of
    cross sections, the generation of hard events and their matching with
    event generators, and the use of a variety of tools relevant to event
    manipulation and analysis.

Homepage: https://launchpad.net/mg5amcnlo

Tags:
    None

Preferred version:
    2.7.3.py3    https://launchpad.net/mg5amcnlo/2.0/2.7.x/+download/MG5_aMC_v2.7.3.py3.tar.gz

Safe versions:
    2.7.3.py3    https://launchpad.net/mg5amcnlo/2.0/2.7.x/+download/MG5_aMC_v2.7.3.py3.tar.gz
    2.7.3.py2    https://launchpad.net/mg5amcnlo/2.0/2.7.x/+download/MG5_aMC_v2.7.3.tar.gz

Variants:
    Name [Default]    Allowed values    Description
    ==============    ==============    ========================================================

    atlas [off]       on, off           Apply changes requested by the ATLAS experimenent on LHC
    collier [off]     on, off           Use external installation of Collier
    ninja [off]       on, off           Use external installation of Ninja

Installation Phases:
    edit    build    install

Build Dependencies:
    collier  fastjet  gosam-contrib  lhapdf  py-six  python  syscalc

Link Dependencies:
    collier  fastjet  gosam-contrib  lhapdf  syscalc

Run Dependencies:
    py-six  python

Virtual Packages:
    None

```